dev/com.ibm.ws.concurrent.persistent_fat_configupdate/publish/servers/com.ibm.ws.concurrent.persistent.fat.configupdate/bootstrap.properties

```diff
@@ -0,0 +1,6 @@
+bootstrap.include=../testports.properties
```

Comment:
bootstrapping.properties files typically have copyrights in them too, you can use this:
```
###############################################################################
# Copyright (c) 2019 IBM Corporation and others.
# All rights reserved. This program and the accompanying materials
# are made available under the terms of the Eclipse Public License v1.0
# which accompanies this distribution, and is available at
# http://www.eclipse.org/legal/epl-v10.html
#
# Contributors:
#     IBM Corporation - initial API and implementation
###############################################################################```